src/ec/eddsa.rs

```diff
@@ -196,8 +198,12 @@ struct Point {
 type Elem = [i32; ELEM_LIMBS];
 const ELEM_LIMBS: usize = 10;
 
-type PublicKey = [u8; ELEM_LEN];
+type PublicKey = [u8; PUBLIC_KEY_LEN];
+const PUBLIC_KEY_LEN: usize = ELEM_LEN;
 
+const PRIVATE_KEY_LEN: usize = ELEM_LEN;
```

Comment:
Addressed in 9f7d94da1feaecb41fe330322ff465073de90905, b5922777d3df7671fc42fd4da8799bc3070237da.